clinica/pydra/engine_utils.py

```diff
@@ -0,0 +1,45 @@
+from pydra import Submitter, Workflow
+
+
+def list_keys(query_bids: dict) -> list:
+    """
+    :query_bids: query dictionary to BIDSDataGrabber
+    :return: IDS of queried elements
+    """
+
+    return list(query_bids.keys())
```

Comment:
I don't think this utility is particularly useful. It's not specific to BIDS (i.e. it works on any dictionary) and it's used once in the code base (`add_input_task`).